### PR TITLE
fix: updated heroku/functions-core to 0.1.4

### PR DESCRIPTION
**Pull request by jeffb-sfdc**
### What does this PR do?
This PR address the issue raised in https://github.com/forcedotcom/salesforcedx-vscode/issues/3677, and updates the functions-core library to 0.1.4

### What issues does this PR fix or reference?
#https://github.com/forcedotcom/salesforcedx-vscode/issues/3677, @https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Uh8DzYAJ/view@

### Functionality Before
Running:
SFDX: Start Function
SFDX: Stop Function
SFDX: Start Function
...resulted in an error.

### Functionality After
Starting & stopping (& starting again) works, and no errors are reported.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "Other"
   ],
   "dependencies": {
-    "@heroku/functions-core": "0.0.7",
+    "@heroku/functions-core": "0.1.4",
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode-core/package.json:28]: Curious why package-lock.json is not being updated (I don't see it getting updated in my local as well).

Comment (jeffb-sfdc) [reply packages/salesforcedx-vscode-core/package.json:28]: I ran `git clean -xfd` (and the also `npm install` and `npm run compile`) and 0.1.4 was pulled down.

**Comment by jeffb-sfdc** [comment]
@jag-sfdc in the commit-workflow job, the publish step is failing (I think due to the CircleCI token).  With the fix you made to publish 53.7.0, was it applied to all (future) builds and releases?  Or just 53.7.0?

**Comment by jag-sfdc** [comment]
I don't think the build failure is related to circleci toke or the marketplace token update I did. This is the error I see in the logs
"ERROR  Command failed: npm list --production --parseable --depth=99999
npm ERR! peer dep missing: chai@>= 2.1.2 < 5, required by chai-as-promised@7.1.1"

**Comment by gbockus-sf** [comment]
This change is no longer value.  Currently used version of @heroku/functions-core is 2.3.

**Pull request closed without merging**